dix/req.c

```diff
@@ -209,11 +209,14 @@ M0_INTERNAL int m0_dix_req_wait(struct m0_dix_req *req, uint64_t states,
 static void dix_req_init(struct m0_dix_req  *req,
 			 struct m0_dix_cli  *cli,
 			 struct m0_sm_group *grp,
+			 int64_t             min_success,
 			 bool                meta)
 {
+	M0_PRE(ergo(min_success < 1, min_success == -1));
```

Comment:
Use M0_DIX_MIN_REPLICA_QUORUM for -1 here.

Reply:
Fixed